homework/java/zhdanok_liudmila/lecture2/task1/src/main/java/by/jrr/mytests/Application.java

```diff
@@ -0,0 +1,12 @@
+package by.jrr.mytests;
+
+
+
+public class Application {
+
+    public static void main(String[] args) throws Exception {
```

Comment:
Here you should place at least comment why main method is blank
Exception not needed